test/testcafe/spec/UnlockAccountChallenge_spec.js

```diff
@@ -70,7 +70,7 @@ const rerenderWidget = ClientFunction((settings) => {
   window.renderPlaygroundWidget(settings);
 });
 
-fixture('Unlock Account');
+fixture('Unlock Account').meta('gen3', false);
```

Comment:
Can you add a comment above this that points to the new file created, just so people know this logic/view is covered.